test/plugins/test_play.py

```diff
@@ -105,6 +105,19 @@ def test_raw(self, open_mock):
 
         open_mock.assert_called_once_with([self.item.path], "echo")
 
+    def test_pls_marker(self, open_mock):
+        self.config["play"]["command"] = (
+            "echo --some params --playlist=$playlist --some-more params"
+        )
+
+        self.run_command("play", "nice")
+
+        open_mock.assert_called_once
+
+        commandstr = open_mock.call_args_list[0][0][1]
+        assert commandstr.startswith("echo --some params --playlist=")
+        assert commandstr.endswith("--some-more params")
```

Comment:
```suggestion
        assert commandstr.endswith(" --some-more params")
```

tiny improvement idea but still you don't really test that the `actual playlist path is in the middle. But probably that would be overkill and introduce more difficulties. I'm fine with just testing start and end.

Reply:
@JOJ0 yes, thanks. I agree with these, no problem. Just had no time to do it yet ...

Reply:
ok please apply the suggestions then. and do another rebase then (I don't seem to be able to do it via the UI, otherwise I'd have to force push your branch) Tell me if that doesn't work and I'll push it :-) Thanks!